### DISCUSSION TITLE
[求解答]假如伴生空洞将零号空洞吞噬后会发生什么

### DISCUSSION THREAD

**Discussion by jiangshixie** [share121/inter-knot#3450] [其他]
![屏幕截图 2024-08-22 090455](https://github.com/user-attachments/assets/c00f12ab-37ad-411a-9530-04431f4aa14a)
如题所问，求各路大神解答

**Comment by lllliufeng** [comment]
会上绳网

**Comment by YolandaWang-0** [comment]
会进入二刺猿的世界哒

**Comment by Fengnix216** [comment]
根据学院学长们的说法，应该是还会有一个零号空洞，只不过这个空洞会挪个窝（
